pom.xml

```diff
@@ -32,12 +32,15 @@
     <invoker.plugin.version>3.1.0</invoker.plugin.version>
     <unpack.build.version>1.0-SNAPSHOT</unpack.build.version>
     <dependency.plugin.version>3.1.2</dependency.plugin.version>
+    <gmavenplus.plugin.version>1.12.1</gmavenplus.plugin.version>
+    <groovy.version>2.5.14</groovy.version>
```

Comment:
do we want to try latest available 3.0.8?

Reply:
Should be possible, http://groovy.github.io/GMavenPlus/dependencies.html states 3.0.7 as test dependency.